python/GafferSceneUI/InteractiveRenderUI.py

```diff
@@ -229,6 +233,100 @@ def __startPauseClicked( self, button ) :
 	def __stopClicked( self, button ) :
 		self.getPlug().setValue( GafferScene.InteractiveRender.State.Stopped )
 
+###############################################################################
+# A widget presenting a summary of the messages in a render nodes messages plug
+###############################################################################
+
+class _MessageSummaryPlugValueWidget( GafferUI.PlugValueWidget ) :
+
+	def __init__( self, plug ) :
+
+		self.__summaryWidget = GafferUI.MessageSummaryWidget( criticalOnly = True, buttonToolTip = "Click to open the render log" )
+		GafferUI.PlugValueWidget.__init__( self, self.__summaryWidget, plug )
+
+		self.__summaryWidget.levelButtonClickedSignal().connect( Gaffer.WeakMethod( self.__levelButtonClicked ), scoped = False )
+
+		self._updateFromPlug()
+
+	def getToolTip( self ) :
+
+		return None
+
+	@GafferUI.LazyMethod( deferUntilPlaybackStops = True )
+	def _updateFromPlug( self ) :
+
+		if self.getPlug() is not None :
+			with self.getContext() :
+				messages = self.getPlug().getValue().value
+		else :
+			messages = Gaffer.Private.IECorePreview.Messages()
+
+		self.__summaryWidget.updateFromMessages( messages )
+
+	def __levelButtonClicked( self, level ) :
+
+		if self.getPlug() is None :
+			return
+
+		window = _MessagesWindow.acquire( self.getPlug() )
+		window.setVisible( True )
+		window.messageWidget().nextMessage( level )
+
+###############################################################################
+# A utility window containing a render nodes message log
+###############################################################################
+
+class _MessagesWindow( GafferUI.Window ) :
+
+	# Use acquire instead to retrieve an existing window or create a new one
+	def __init__( self, parentWindow, plug ) :
+
+		GafferUI.Window.__init__( self, borderWidth = 8 )
+
+		self.setChild( _MessagesPlugValueWidget( plug ) )
+
+		self.setSizeMode( GafferUI.Window.SizeMode.Manual )
```

Comment:
`Manual` appears to be the default size mode, so is this redundant?

Reply:
'sgone.